packages/api/get-stops.ts

```diff
@@ -71,13 +71,27 @@ export const getQueryInput = (event: APIGatewayEvent): StopsQueryInput => {
         .filter((stop) => stop)
         .map((stop) => stop.trim());
 
+    const busStopType = queryStringParameters?.busStopType || "";
+    const busStopTypeArray = busStopType
+        .split(",")
```

Comment:
is it worth catering to uppercase input by lowering the case of the query strings when I put CUS no results came up but cus works

Reply:
I've changed the code so we only accept uppercase queries instead of lowercase.

Speaking to Laurence, we don't want to allow lowercase inputs at all.